fairlearn/reductions/_moments/moment.py

```diff
@@ -25,31 +24,50 @@ class Moment:
     def __init__(self):
         self.data_loaded = False
 
-    def load_data(self, X, y, **kwargs):
+    def load_data(self,
+                  X,
+                  y: pd.Series,
```

Comment:
okay for now. in future, this should be the same as `y_true` and `y_pred` in `MetricFrame`

Reply:
I believe that by the time this call is made, `y` should have been turned into a `Series` already. Having each method have to accept all types of input feels like a good way of doing a lot of unnecessary conversions.

Reply:
agreed...